src/js/utils/networks.js

```diff
@@ -615,7 +615,7 @@ export const Tokens = {
   'rinkeby': [
     {
       symbol: 'ETH',
-      name: 'ETH',
+      name: 'Ethereum',
```

Comment:
Not sure, but this might cause Coin Gecko issues.
Long story short, I use the symbol and name to check the coin gecko ID and that's why I changed some of the names to the Gecko ones. That's because I use a binary search instead of a full search (O(log(n)) instead of O(n))

Anyway, to test, just reset your cache and check if the price for ETH works (check console too)

Reply:
Looks like it works with 'Ethereum'

Reply:
Good